log.py

```diff
@@ -52,7 +53,45 @@ def format(self, record):
             data = data.encode("utf8")
         return data
 
-       
+class SysLogger(object):
+
+    NAME = 'sysLog'
+
+    JSON_LOG_FORMAT = 'json'
+    TEXT_LOG_FORMAT = 'text'
+
+    # Settings for the integration with protocol=INTERNAL_LOGGING
+    LOG_FORMAT = 'log_format'
+    LOG_MESSAGE_TEMPLATE = 'message_template'
+
+    SETTINGS = [
+        { "key": LOG_FORMAT, "label": _("Log Format"), "type": "select",
+            "options": [
+                { "key": JSON_LOG_FORMAT, "label": _("json") },
+                { "key": TEXT_LOG_FORMAT, "label": _("text") }
+            ]
+        },
+        { "key": LOG_MESSAGE_TEMPLATE, "label": _("template") }
```

Comment:
Maybe add a description that links to documentation on the message template format?